test/decl/var/lazy_properties.swift

```diff
@@ -29,7 +29,7 @@ class TestClass {
 
   lazy var d : Int  // expected-error {{lazy properties must have an initializer}} {{3-8=}}
 
-  lazy var (e, f) = (1,2)  // expected-error {{'lazy' cannot destructure an initializer}} {{3-8=}}
+  lazy var (e, f) = (1,2)  // expected-error 2{{'lazy' cannot destructure an initializer}} {{3-8=}}
```

Comment:
Why is this being emitted redundantly?

Reply:
The old diagnostic would delete the attribute so it was only emitted once. Now it's emitted for both `e` and `f`.

Reply:
Can we diagnose it when processing the `PatternBindingDecl` somehow?